assets/js/data/cart/actions.ts

```diff
@@ -73,26 +113,31 @@ export function receiveRemovingCoupon( couponCode ) {
 /**
  * Returns an action object for updating a single cart item in the store.
  *
- * @param {Object}   [response={}]    A cart item API response.
- * @return {Object} Object for action.
+ * @param  {CartResponseItem} [response=null] A cart item API response.
+ * @return {ResponseCartItemAction}            Object for action.
  */
-export function receiveCartItem( response = {} ) {
+export function receiveCartItem(
+	response: CartResponseItem | null = null
+): ResponseCartItemAction {
 	return {
 		type: types.RECEIVE_CART_ITEM,
 		cartItem: response,
 	};
 }
```

Comment:
I find with action creators specifically, it's nice to let TypeScript infer the types. They just return simple objects and it doesn't seem helpful to describe the shape twice. Adding the _`const` assertion_ will make sure the return type has exactly the shape we expect:

```suggestion
export function receiveCartItem(
	response: CartResponseItem | null = null
) {
	return {
		type: types.RECEIVE_CART_ITEM,
		cartItem: response,
	} as const;
}
```

Reply:
done in d2c17a7